products/gerrit.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Gerrit
+category: server-app
+iconSlug: "gerrit"
+permalink: /gerrit
+versionCommand: "java -jar gerrit.war version"
+releasePolicyLink: https://www.gerritcodereview.com/support.html
+changelogTemplate: https://www.gerritcodereview.com/__RELEASE_CYCLE__.html#{{"__LATEST__"| replace:'.',''}}
+activeSupportColumn: false
+eolColumn: "Community Support"
```

Comment:
```suggestion
eolColumn: Security Support
```

While this is also best-effort, expectation of security updates for last 3 releases is the norm here. Security updates t is the most important thing most readers are concerned about, and using "Community Support" is confusing, especially since we aren't explaining it in the text.

Reply:
I changed it to "Support", we do not only provide security fixes for the last 3 releases